docker/Dockerfile.aarch64-unknown-linux-gnu

```diff
@@ -12,10 +12,11 @@ RUN /xargo.sh
 
 FROM cross-base as build
 
-RUN apt-get update && apt-get install --assume-yes --no-install-recommends \
-    g++-aarch64-linux-gnu \
-    gfortran-aarch64-linux-gnu \
-    libc6-dev-arm64-cross
+ENV CROSS_TOOLCHAIN_PREFIX=aarch64-linux-gnu-
+ENV CROSS_SYSROOT=/usr/aarch64-linux-gnu
```

Comment:
I decided to move these variable above `apt-cross-essential.sh`, since I wanted to use `CROSS_TOOLCHAIN_PREFIX` to test for pre-installed gfortran (which was causing some headaches).